viscm/gui.py

```diff
@@ -16,14 +16,11 @@
 # Do this first before any other matplotlib imports, to force matplotlib to
 # use a Qt backend
 from matplotlib.backends.qt_compat import QtWidgets, QtCore, QtGui, _getSaveFileName
-from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas4
-from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas5
 
-def FigureCanvas(fig):
-    try:
-        return FigureCanvas5(fig)
-    except Exception:
-        return FigureCanvas4(fig)
+try:
+    from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
+except Exception:
+    from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
```

Comment:
```suggestion
try:
    from matplotlib.backends.backend_qtagg import FigureCanvasQTAgg as FigureCanvas
except ImportError:
    try:
        from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
    except ImportError:
        from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
```

Reply:
This will make it work with Qt5 or Qt6 (and presumably with Qt7 when it comes as we intend to stick with a single Qt backend modules).